File: backend/calApp/calorie-service/src/main/java/com/caliq/calorie_service/models/dto/CalendarResponse.java
package com.caliq.calorie_service.models.dto;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.List;

public record CalendarResponse(List<CaloryDays> calendar) {

    public record CaloryDays(LocalDate date,
                             BigDecimal caloryNorm,
                             BigDecimal caloryCons){}
}

File: androidApp/NytriSync/app/src/main/java/com/ga3t/nytrisync/data/model/CalendarResponse.kt
package com.ga3t.nytrisync.data.model
import java.math.BigDecimal
data class CalendarResponse(
    val calendar: List<CaloryDays>
) {
    data class CaloryDays(
        val date: String,
        val caloryNorm: BigDecimal,
        val caloryCons: BigDecimal
    )
}